lib/tools.py

```diff
@@ -328,9 +335,12 @@ def sort_csv(filename):
     infields = next(csv_reader_infile)
 
     #  write the data to a temporary file and sort it
-    file_temp = open(os.path.normpath("tempfile"), 'wb')
+    file_temp = open(os.path.normpath("tempfile"), 'w')
 
-    csv_writer = csv.writer(file_temp, dialect='excel', escapechar='\\')
+    if os.name == 'nt':
```

Comment:
Doesn't work. At the moment I think this issue is pertinent to python3, so I think a separate issue (if needed) can be created. Although I'm totally ok with having Carriage returns in the file.
@ethanwhite